argclass.py

```diff
@@ -269,7 +269,12 @@ class AbstractGroup:
 
 
 class AbstractParser:
-    pass
+    __parent__: Optional["AbstractParser"] = None
+
+    def _get_chain(self) -> Tuple["AbstractParser", ...]:
+        if self.__parent__ is None:
+            return (self,)
+        return (self,) + self.__parent__._get_chain()
```

Comment:
Can we use more pythonic way for this? Iterators may be to prevent create tuple on every chain? 
```(python)
from dataclasses import dataclass
import typing

@dataclass
class Chain:
    __parent__: typing.Optional['Chain'] = None

     def __iter__(self):
         yield self
         if self.__parent__ is None:
             return
         yield from self.__parent__

# a = Chain()
# b = Chain(__parent__=a)
# tuple(b)
```

Reply:
More abstractions are needed!)
I don't see this as a big problem, because it is used at the start of the application and does not consume megabytes of memory
But it solves the real problem)

Reply:
I sure think somebody (@mosquito) will build them in runtime